src/app/components/progress-indicator/passive-progress-indicator/index.html

```diff
@@ -0,0 +1,276 @@
+<sky-tabbed-demo-page
+  pageTitle="Passive Progress Indicator"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The progress indicator module visually represents progress through sequential steps toward a goal. Passive progress indicators represent steps outside of user control.
+    </p>
+  </sky-demo-page-summary>
+
+  <div class="sky-margin-stacked-content-section">
+    <sky-demo-page-example>
+      <sky-progress-indicator-demo>
+      </sky-progress-indicator-demo>
+    </sky-demo-page-example>
+  </div>
+
+  <sky-tabset id="content-tabs">
+    
+    <sky-tab tabHeading="Design">
+      <sky-demo-page-content sectionHeading="Usage">
+        <h3 class="sky-text-success">Use when</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Use passive progress indicators when users must be aware of sequential progress that they are not responsible for.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div>
+              <img src="~/assets/img/guidelines/progress-indicator/passive-progress-usage-1.png" />
+              <div class="caption caption-success sky-text-success">Do use passive progress indicators to track sequential steps that other users take.</div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+    
+        <h3 class="sky-text-danger">Don't use when</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Don't use passive progress indicators for tasks that require user interaction.</p>
```

Comment:
Should we say to use waterfall indicators or wizards instead and link to those topics?

Reply:
Agreed.